ios/3.39.0/Update change log and version number (#808)
MapsPeople/docs: src/changelogs/ios/v3/CHANGELOG.md

### Removed
-->

## [3.39.0] 2022-03-31

### __*Note: Due to [a bug in CocoaPods](https://github.com/CocoaPods/CocoaPods/issues/7155) it is necessary to include the post_install hook in your Podfile described in the [PodFile post_install](https://github.com/MapsIndoors/MapsIndoorsIOS/wiki/Podfile-post_install) wiki*__

### Added

- We added an enum for `MPHighwayType`, so it is now easy to know which values are valid to use.
- We added the possibility to get the `translatedName` property of Location Types.
- We added the method `MPVenue.hasGraph` so it can be checked if there is a routing network available on a solution to get directions.

### Changed

> - A number of headers are no longer public – they were not intended for public consumption anyways…

Reviewer: ```suggestion
- A number of headers have been made private.
```
